recipes/virsorter/build.sh

```diff
@@ -18,7 +18,17 @@ make
 
 ### Organizing executables
 cd ../
+
 PKG_OUTDIR="${PREFIX}"/share/"${PKG_NAME}"-"${PKG_VERSION}"-"${PKG_BUILDNUM}"/
 mkdir -pv "${PKG_OUTDIR}"/ "${PREFIX}"/bin/
 cp -r $(ls | grep -v "conda" | grep -v "build") "${PKG_OUTDIR}"/
+
 ln -s "${PKG_OUTDIR}"/wrapper_phage_contigs_sorter_iPlant.pl "${PREFIX}"/bin/
+ln -s "${PKG_OUTDIR}"/Scripts/ "${PREFIX}"/bin/Scripts
+
+
+
+### Fixing permissions
+
+chmod 644 "${PKG_OUTDIR}"/Scripts/Makefile "${PKG_OUTDIR}"/Scripts/Sliding_windows_3.c "${PKG_OUTDIR}"/Scripts/Sliding_windows_3.o
```

Comment:
Should those not be deleted? Likewise, there are a bunch of random files under `share/virsorter-1.0.5-3` that can presumably be removed (cpanfile, Dockerfile, LICENSE, Makefile, and README.md).

Reply:
Good point. I cleaned it all up and rebuilt it so it should be ready to go. Thanks for your help!